internal/cluster/transition_test.go

```diff
@@ -3640,7 +3640,9 @@ var _ = Describe("Ocs Operator use-cases", func() {
 		ctrl                                          *gomock.Controller
 		dbName                                        string = "cluster_transition_test_with_ocs_validations"
 	)
-
+	mockIsValidMasterCandidate := func() {
```

Comment:
Why is this required?

Reply:
If the role is autoassign we need to mock the IsValidMasterCandidate as defined in the interface